### PR TITLE
Rename over'/traverseOf' -> overU/traverseOfU

### PR DESCRIPTION
**Pull request by aspiwack**
It's more in line with the rest of linear-base. In an ideal word we
don't even need these function and over & traverse are multiplicity
polymorphic. But we are not there yet.

Depends on #219 #221 .

### CODE REVIEW

**Comment by aspiwack** [comment]
There is a bit of noise due to the dependency on two different PRs, so either review just the last commit, or wait until #219 is merged.

**Review by utdemir — APPROVED** [review]

**Review by Divesh-Otwani — APPROVED** [review]

**Pull request merged**
